### DISCUSSION TITLE
Passing record to custom field in a view page

### DISCUSSION THREAD

**Discussion by BasharAl-Mohammad** [filamentphp/filament#5342] [Help]
Hello,
I am working on an e-commerce project, I have a section in my dashboard where the `User` has to see the `Orders`. I couldn't gather all the needed data through filament components and I want to create a custom view page.

So I created a view page `ViewOrder` and I noticed that the view page takes form fields and disables them. So I created a custom field and I'm trying to pass the record from the `OrderResource` to the field but $this->record doesn't work.

```
DescriptionList::make('overview')
                    ->order(function (Callable $get){
                        dd($get('id'));
                    }),
```

My question is how can I access the model record and pass them to the custom fields/field?

**Comment by ArtMin96** [comment]
Maybe this will help. [https://filamentphp.com/tricks/getting-other-fields-values-in-viewfield-blade-file](https://filamentphp.com/tricks/getting-other-fields-values-in-viewfield-blade-file)

**Reply by BasharAl-Mohammad** [reply]
I created a custom page instead of a view page. I managed to pass the order eloquent and all relations needed. Appreciate your help buddy.
